cmd/kpromo/cmd/pr/pr.go

```diff
@@ -135,6 +136,14 @@ func init() {
 		"interactive mode, asks before every step",
 	)
 
+	PRCmd.PersistentFlags().StringSliceVarP(
+		&promoteOpts.images,
+		"image",
```

Comment:
I don't have data points about the most frequent behavior is, but I'm wondering if we should have skip-images instead (default all, skip what you don't want to promote vs current implementation default all, explicitly list all you 
want to promote)

Reply:
I switched to explicitly listing images to include based on @justaugustus's comment [here](https://github.com/kubernetes-sigs/promo-tools/issues/479#issuecomment-1006216736)

Reply:
thanks for the context! then +1 from me

Reply:
ref: https://github.com/kubernetes-sigs/promo-tools/issues/479#issuecomment-1006200436

@CecileRobertMichon @fabriziopandini -- I expanded a little on this (inclusion vs exclusion) in the [2022-01-18 RelEng meeting](https://docs.google.com/document/d/16GqCjnEh86w8yADcrUylNoE1y1sqjIMYNC_gdk5WPSQ/edit#bookmark=id.lzc56fdhbbvl)...

Inclusion: It's very likely that you will know exactly what images you're interested in promoting.
Exclusion: You will need to discover which images you need to exclude. While you may know some of them, at any point new images could be created within the staging repository (maybe by error or by another maintainer who stages their images in the repo) which match your exclusion tag.